### DISCUSSION TITLE
[Feature] Futuristic Feature: editing picture dates based on age predictions of person

### DISCUSSION THREAD

**Discussion by diyoyo** [immich-app/immich#6008] [Feature Request]
### The feature

This would be my dream feature: being able to reassign dates based on age classification of a given person.

Case 1:
You take an old paper-based album from your grand-mother and start scanning it. It has all the pictures of your father growing up, but the dates are missing. The attributed date is the date of scanning. Then, you make sure your father is tagged at all different ages. Since you know the birth date, and the latest age of the most recent pic available, you may be able to predict the ages in between, and be able to reassign dates...

Case 2:
You have plenty of pictures of your baby child. Growing up, other babies start showing up in the pics. On the Family Whatsapp, people start sharing Memories of the first baby at the same age of the second baby. This means : Untagged, compressed, date-removed, GPS-removed copies of the pictures you already have in your library. These pictures are showing up in between other pictures of the now-grown-up kid, in the Face album. It would be nice to be able to reassign these pics to the correct date, based on clustering with the original ones.

I guess this is a bit tricky, but it's an awesome feature, imho

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by aviv926** [comment]
duplicate of https://github.com/immich-app/immich/discussions/5133
Actually it's something I'm working on, almost finished and perfect
But it will take time before I do a PR
